src/main/scala/cromwell/binding/FinalCall.scala

```diff
@@ -0,0 +1,38 @@
+package cromwell.binding
```

Comment:
Specifically this seems to me like "how cromwell handles WDL" and not "what is wdl"